cumulusci/tasks/bulkdata.py

```diff
@@ -807,14 +926,24 @@ def _download_file(uri, bulk_api):
         yield f
 
 
-def process_incoming_rows(f, record_type=None):
+isoformat_regex = r'"2[0-9]{3}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}.[0-9]{3}Z"'
+
+
+def process_incoming_rows(f, record_type=None, relative_date_now=None):
     if record_type and not isinstance(record_type, bytes):
         record_type = record_type.encode("utf-8")
     for line in f:
         if record_type:
-            yield line.rstrip() + b"," + record_type + b"\n"
-        else:
-            yield line
+            line = line.rstrip() + b"," + record_type + b"\n"
+        if relative_date_now:
+            line = line.decode("utf8")
+            for isoformat in re.findall(isoformat_regex, line):
```

Comment:
Shouldn't we target only the columns that were marked as dates? If there's a text field that happens to have an ISO format date in it, we wouldn't want to turn that into an integer.

Reply:
I need to think about how to implement that.  This implementation doesn't require awareness of the position on columns in rows.  Supporting only columns specified as relative_date columns would require passing more args to this method.  I'll look into it though.